styles/tokens/spacing.css

```diff
@@ -6,4 +6,8 @@
   --diamond-spacing-sm: calc(var(--diamond-spacing) / 2);
   --diamond-spacing-lg: calc(var(--diamond-spacing) * 2);
   --diamond-spacing-xl: calc(var(--diamond-spacing) * 4);
+
+  --diamond-spacing-fluid: clamp(1rem, 5vw, 4rem);
+  --diamond-spacing-fluid-sm: clamp(1rem, 5vw, 3rem);
+  --diamond-spacing-fluid-lg: clamp(1rem, 5vw, 5rem);
```

Comment:
Should section support these?

Reply:
Yeah, probably card too looking at EtchOS